ui/components/multichain/pages/page/index.scss

```diff
@@ -1,5 +1,44 @@
+@use "design-system";
+
 @import 'components/';
 
+.multichain-app-header-logo + .multichain-page {
+  $height-screen-sm-min: 720px;
+
+  @media screen and (min-width: design-system.$screen-sm-min) and (min-height: $height-screen-sm-min) {
+    padding-top: 0;
+  }
+}
+
 .multichain-page {
   // TODO: Add page sizes
```

Comment:
Is this TODO still relevant?

Reply:
Good question. I left a comment in the last PR mentioning I'll remove if someone else agrees with me we can remove this. Removed it now and can add back in if others find it still relevant https://github.com/MetaMask/metamask-extension/pull/31454/commits/7ea8316488480abe732cf7b9ac80755422e4b7eb cc: @georgewrmarshall 

thanks for asking!